src/wmrc/main.py

```diff
@@ -187,84 +199,424 @@ def process():
+                columns={
+                    "longitude_": "longitude",
+                    "accept_material_dropped_off_by_the_public": "accept_material_dropped_off_by_",
+                    "tons_of_material_diverted_from_landfills_last_year": "tons_of_material_diverted_from_",
+                }
+            )
+        )
+        renamed_df["id_"] = renamed_df["id_"].astype(str)
+
+        return renamed_df
+
+    @staticmethod
+    def _get_county_names(input_df: pd.DataFrame, gis: arcgis.gis.GIS) -> pd.DataFrame:
+        """Assigns a county name to each facility based on the SGID county boundaries hosted in AGOL.
+
+        Args:
+            input_df (pd.DataFrame): Facility dataframe with "latitude" and "longitude" columns
+            gis (arcgis.gis.GIS): AGOL org containing the county boundaries
+
+        Returns:
+            pd.DataFrame: A spatially-enabled dataframe of the facilities in WKID 26912 with county names added.
+        """
+
+        #: Load counties from open data feature service
+        counties_df = pd.DataFrame.spatial.from_layer(
+            arcgis.features.FeatureLayer.fromitem(gis.content.get(config.COUNTIES_ITEMID))
+        )
+        counties_df.spatial.project(26912)
+        counties_df.reset_index(inplace=True)
+        counties_df = counties_df.reindex(columns=["SHAPE", "NAME"])  #: We only care about the county name
+        counties_df.spatial.set_geometry("SHAPE")
+        counties_df.spatial.sr = {"wkid": 26912}
+
+        #: Convert dataframe to spatial
+        spatial_df = pd.DataFrame.spatial.from_xy(input_df, x_column="longitude", y_column="latitude")
+        spatial_df.reset_index(drop=True, inplace=True)
+        spatial_df.spatial.project(26912)
+        spatial_df.spatial.set_geometry("SHAPE")
+        spatial_df.spatial.sr = {"wkid": 26912}
+
+        #: Perform the join, clean up the output
+        joined_points_df = spatial_df.spatial.join(counties_df, "left", "within")
+        joined_points_df.drop(columns=["index_right"], inplace=True)
+        joined_points_df.rename(columns={"NAME": "county_name"}, inplace=True)
+        joined_points_df["county_name"] = joined_points_df["county_name"].str.title()
+
+        return joined_points_df
+
+    def _load_salesforce_data(self) -> helpers.SalesForceRecords:
+        """Helper method to connect to and load data from Salesforce.
+
+        Returns:
+            helpers.SalesForceRecords: An object containing the records from Salesforce along with other derived data.
+        """
+
+        salesforce_credentials = extract.SalesforceApiUserCredentials(
+            self.secrets.SF_CLIENT_SECRET, self.secrets.SF_CLIENT_ID
+        )
+        salesforce_extractor = extract.SalesforceRestLoader(self.secrets.SF_ORG, salesforce_credentials)
+
+        salesforce_records = helpers.SalesForceRecords(salesforce_extractor)
+        salesforce_records.extract_data_from_salesforce()
+
+        return salesforce_records
+
+
+class Summarize:
```

Comment:
**question**: if every method is static, is the class used for organization? Would it be simpler to import a summarize.py file?

Reply:
Good catch, that would be simpler.